just_bin_it/histograms/histogram2d_map.py

```diff
@@ -105,12 +100,11 @@ def data(self):
     def shape(self):
         return self.width, self.height
 
-    def add_data(self, pulse_time, tofs, det_ids, source=""):
+    def add_data(self, pulse_time, det_ids, source=""):
```

Comment:
Cannot do this as the signature needs to match the add_data methods in the other histogram types.

Reply:
it's used in `Histogrammer` class where instance is checked for `DetHistogram` and appropriate `add_data` with proper signature is used. May be not a good idea. we were not sure,

Reply:
Not a good idea, better to have the differences abstracted away.